shell/initialize/app-extended.js

```diff
@@ -1,5 +1,6 @@
 // Taken from @nuxt/vue-app/template/index.js
 // This file was generated during Nuxt migration
+// TODO: Add more clarification to this comment
```

Comment:
There shouldn't be any TODOs in code. 

The code comes from nuxt, aka https://github.com/nuxt/nuxt/blob/v2.18.1/packages/vue-app/template/index.js --> which is the old node_modules/@nuxt/vue-app/template/index.js

Reply:
You cannot just put a comment with a path, you need to add an explanation and something working.
Thanks for the link, that's better and I can add it 👍

Reply:
The path comes from the node package, hence the @nuxt. Ideally i was trying to avoid a direct reference. 

Can we just use `node_modules/@nuxt/vue-app/template/index.js`?

Reply:
would you mind to motivate the choice, so I can understand as well and we can find an agreement?

Reply:
Can you leave unchanged and create an issue?